src/main/development.ts

```diff
@@ -1,6 +1,7 @@
+/* eslint-disable @typescript-eslint/no-require-imports */
 import * as express from 'express';
 
-const setupDev = (app: express.Express, developmentMode: boolean): void => {
+export const setupDev = (app: express.Express, developmentMode: boolean): void => {
```

Comment:
Do we still need module.exports at the bottom now we export the const here?

Reply:
Good spot, updated